include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -55,6 +65,7 @@ ORT_DEFINE_RELEASE(TensorTypeAndShapeInfo);
 ORT_DEFINE_RELEASE(TypeInfo);
 ORT_DEFINE_RELEASE(Value);
 
+// This is used internally by the C++ API. This is the common base class used by the wrapper objects.
```

Comment:
> This is used internally by the C++ API [](start = 2, length = 39)

This can not be internal bc publicly used classes inherit from it.

Reply:
Oh.. it's internal to the header file, it's not internal to our library. The header file depends on it, but users of the header file don't need to pay much attention to it. Is there a better phrasing I could use?